PeerioTechnologies/peerio-desktop
# feat: new pdf generator

testing:
- create a new account, save account key pdf
- go to security settings, save another account key pdf
- make sure both pdfs match the mocks and contain correct data
- make sure pdfs render correctly in different apps (browsers, reader, preview) and platforms (mac, linux, windows)
- make sure you can copy/paste account key and username and login

sdk branch:
https://github.com/PeerioTechnologies/peerio-icebear/pull/279
